backend/tests/src/tests/json-schemas/user/user-by-id/user-by-id-response.schema.ts

```diff
@@ -0,0 +1,26 @@
+import { permissionSchema } from '../../json-schemas';
+
+const userSchema = {
```

Comment:
In Swagger this dto was recently added to types. And also there will be same type, but without the permission array.
So better rename the schema to `userWithPermissionsSchema`,
`json-schemas/user/user-by-id/user-by-id-response.schema.ts` to `user-with-permissions-response.schema.ts`,
`json-schemas/user/user-by-id/user-by-id.ts` to `user-with-permissions.ts`,
and `json-schemas/user/user-by-id` to `user-with-permissions`